src/index.ts

```diff
@@ -442,7 +442,8 @@ export function graphqlHTTP(options: Options): Middleware {
 
     if (isAsyncIterable(executeResult)) {
       response.setHeader('Content-Type', 'multipart/mixed; boundary="-"');
-      sendPartialResponse(pretty, response, formattedResult);
+      // There will always be a next, either a data payload, or an error
+      sendPartialResponse(pretty, response, formattedResult, true, true);
```

Comment:
Can you send the initial boundary before calling `sendPartialResponse`, so a flag for `initial` isn't needed?

Reply:
Yeah that is probably better.